operator/roles/kiali-deploy/tasks/openshift/os-get-kiali-route-url.yml

```diff
@@ -13,43 +15,35 @@
   delay: 10
   when:
   - is_openshift == True
-  - kiali_vars.auth.strategy == "openshift"
 
 - name: Set Kiali TLS Termination from OpenShift route
   set_fact:
     kiali_route_tls_termination: "{{ kiali_route_raw['resources'][0]['spec']['tls']['termination'] }}"
   when:
   - is_openshift == True
-  - kiali_vars.auth.strategy == "openshift"
 
 - name: Detect HTTP Kiali OpenShift route protocol
   set_fact:
     kiali_route_protocol: "http"
   when:
   - is_openshift == True
-  - kiali_vars.auth.strategy == "openshift"
   - kiali_route_tls_termination == ""
 
 - name: Detect HTTPS Kiali OpenShift route protocol
   set_fact:
     kiali_route_protocol: "https"
   when:
   - is_openshift == True
-  - kiali_vars.auth.strategy == "openshift"
   - kiali_route_tls_termination != ""
 
 - name: Create URL for Kiali OpenShift route
   set_fact:
     kiali_route_url: "{{ kiali_route_protocol }}://{{ kiali_route_raw['resources'][0]['status']['ingress'][0]['host'] }}"
   when:
   - is_openshift == True
-  - kiali_vars.auth.strategy == "openshift"
-
-# Now that we have the kiali_route_url we can process the oauth.yaml
 
-- name: Process OpenShift OAuth client
+- name: Process OpenShift Console Links
```

Comment:
this is in os-main.yml now - all that this os-get-kiali-route-url.yml should do is ultimately set the kiali_route_url fact. So lines 45 and following should be deleted.

Reply:
Good catch, thanks.